fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -3534,13 +3534,6 @@ private void createTaskForMaterializedView(String dbName, MaterializedView mater
             if (optHints != null) {
                 Map<String, String> taskProperties = task.getProperties();
                 taskProperties.putAll(optHints);
-                if (materializedView.getWarehouseId() != WarehouseManager.DEFAULT_WAREHOUSE_ID) {
-                    taskProperties.put(PropertyAnalyzer.PROPERTIES_WAREHOUSE_ID,
-                            String.valueOf(materializedView.getWarehouseId()));
-
-                    LOG.debug("set warehouse {} in createTaskForMaterializedView",
-                            materializedView.getWarehouseId());
-                }
```

Comment:
I don't quite understand the working principle of MV. Why can this part of the logic be deleted?
If so, there will no such property in task created by MV.